### ISSUE TITLE
Any way to disable the crash sound?

### ISSUE THREAD

**Issue by OrangeFlavored** [Kaldaien/SpecialK#1]
Was playing Valkyria Chronicles with the global injector and the game crashed for some reason, causing that alert sound to play very, very loud through my headphones (I guess that game's volume is low in general so I had my amp cranked up to compensate) and it scared the crap out of me!  I don't see much reason for a sound to play just to let me know a game crashed anyway so is there some way I can disable this in the future?

**Comment by Idearum** [comment]
```
;UseCrashHandler=true plays MGS alert sound on crashes and outputs crash info to logs/crash/<timestamp>/*.log
UseCrashHandler=true
```

Basically open the SpecialK.ini file for the game (or d3d9.ini or whatever it's called) and change UseCrashHandler to false.

**Comment by Nicholas-Steel** [comment]
Wouldn't that disable all crash handling and not just the sound?

**Comment by Idearum** [comment]
Special K does not have a parameter that controls the sound itself, so the only way to disable the sound is to disable the crash handler entirely.

The crash handler of Special K is basically a component that attempts to capture additional diagnostics in case of a crash -- *any* crash that the game experiences, regardless of cause or fault. Disabling it doesn't really result in anything else than that Special K does not bother to create a crash.log file when a game experiences a crash.

So basically no functionality is lost by disabling it when dealing with an already fully functional game that have just crashed once or twice or so. The crash handler is mostly useful when troubleshooting continuous and repeatable crashes that Special K itself is involved in.
